Workbooks/UpdateCompliance/UpdateComplianceHub/Update Compliance Hub.workbook

```diff
@@ -0,0 +1,1660 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "crossComponentResources": [
+          "{mappedSubscription}"
+        ],
+        "parameters": [
+          {
+            "id": "67dcb2cb-14b1-4d77-b52b-f728abbf43a2",
+            "version": "KqlParameterItem/1.0",
+            "name": "mappedTenantId",
+            "type": 1,
+            "isGlobal": true,
```

Comment:
All of the parameters in this section are global, is that necessary? parameters only need to be global if something downstream of this step rewrites the value. Otherwise if alll the steps below just read the value, this doesn't have to be global.

Declaring many parameters global can impact performance